xapian-core/tests/api_cluster.cc

```diff
@@ -29,14 +29,38 @@
 #include "testsuite.h"
 #include "testutils.h"
 
+static void
+make_generated_db(Xapian::WritableDatabase &db, const std::string &)
+{
+    const std::unordered_set<std::string>
+    test_strings({"This line is about a cluster. Cluster is important and is everywhere",
+		  "We need to search for special cluster. Cluster cluster cluster",
+		  "Computer cluster is a special example of a cluster. Used to search fast",
+		  "Another example of cluster is a star cluster. Star cluster has a lot of stars"});
```

Comment:
> (The first const attaches to the char *, meaning that the elements of the array are pointers to constant character data; the second const means the pointers in the array are also constant).

This is something new to me. I never knew we could use the keyword twice in once declaration. Thanks for this.